src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -744,14 +744,17 @@ export const ComputeModalBase = ({
         [Utils.DEFAULT, () => runtimeTypes.gceVm] // for when there's no existing runtime
       )
 
+      const diskSize = Utils.cond([!!runtimeConfig?.diskSize, () => runtimeConfig.diskSize],
```

Comment:
Be wary coercing numbers to booleans. 0 is falsy, but is 0 a possible value for `runtimeConfig.diskSize` or `runtimeConfig.masterDiskSize`?

Reply:
Also, formatting nit: move the argument to the next line so all conditions are aligned.

```suggestion
      const diskSize = Utils.cond(
        [!!runtimeConfig?.diskSize, () => runtimeConfig.diskSize],
```

Reply:
Disk sizes can never be 0